tests/unit/remote/ssh/test_ssh.py

```diff
@@ -30,15 +30,17 @@ def test_url(self):
         path = "/path/to/dir"
 
         # URL ssh://[user@]host.xz[:port]/path
-        url = "ssh://{}@{}:{}{}".format(user, host, port, path)
+        if os.name == "nt":
```

Comment:
is this still necessary?

Reply:
Yes. It is necessary as I answered in above comment. In windows, we need an indicator to separate between `port` and `path`. If not the `port` will get `c` drive character and the make the port validation failed as it contain a character in it.

Reply:
And now this is no longer needed, right?

Reply:
Oh, You read my mind. I was pushed new update while you posted the above comment :)
DONE